controllers/comments.js

```diff
@@ -42,14 +40,12 @@ module.exports = {
     try {
       // Find comment by id
       let comment = await Comment.findById({ _id: req.params.id });
+      let event = comment.event
       console.log(comment)
-      // FInd post by id
-      let event = await Event.findById({ _id: req.params.id });
```

Comment:
req.params.id referes to the id of the comment. We create a back-reference to the event from the comment on line 43 and pass that as the redirect parameter.